lib/metasploit/framework/login_scanner/ivanti_login.rb

```diff
@@ -0,0 +1,100 @@
+require 'metasploit/framework/login_scanner/http'
+
+module Metasploit
+  module Framework
+    module LoginScanner
+      class Ivanti < HTTP
+
+        def create_user_request(username, password, protocol, peer)
+          {
+            'method' => 'POST',
+            'uri' => normalize_uri('/dana-na/auth/url_default/login.cgi'),
+            'ctype' => 'application/x-www-form-urlencoded',
+            'headers' =>
+            {
+        'Cache-Control'=> 'max-age=0',
+        'Sec-Ch-Ua'=> '"Chromium";v="131", "Not_A Brand";v="24"',
+        'Sec-Ch-Ua-Mobile'=> '?0',
+        'Sec-Ch-Ua-Platform'=> 'Linux',
+        'Accept-Language'=> 'en-US,en;q=0.9',
+        'Origin'=> "#{protocol}://#{peer}",
+        'Upgrade-Insecure-Requests'=> '1',
+        'Accept'=> 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7',
+        'Sec-Fetch-Site'=> 'same-origin',
+        'Sec-Fetch-Mode'=> 'navigate',
+        'Sec-Fetch-User'=> '?1',
+        'Sec-Fetch-Dest'=> 'document',
+        'Referer'=> "#{protocol}://#{peer}/dana-na/auth/url_default/welcome.cgi",
+        'Accept-Encoding'=> 'gzip, deflate, br',
+        'Priority'=> 'u=0, i'
```

Comment:
Are these headers needed? I don't think we normally have these kinds of headers in the login modules 👀